server/server/search/tests/test_instant_search.py

```diff
@@ -120,4 +124,52 @@ def test_normalize_string():
     assert normalize_string('rupa') == 'rupa'
     assert normalize_string('Rūpā') == 'Rupa'
     assert normalize_string('samma') == 'samma'
-    assert normalize_string('sāmma') == 'samma'
\ No newline at end of file
+    assert normalize_string('sāmma') == 'samma'
+
+
+def instant_search(query, languages):
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (testing):** Missing test case for partial match query parameter.

The test suite includes a parameter for `matchpartial=false` in the query, but there's no test case exploring the behavior when `matchpartial=true`. Adding such a test could ensure the feature works as expected across different configurations.

```suggestion
def test_matchpartial_true_search():
    instant_search('volpage:s ii', ['en', 'pli'], matchpartial=True)
```